Microsoft.Toolkit.Uwp/Commands/DelegateCommand.cs

```diff
@@ -14,16 +14,16 @@
 using System.Diagnostics;
 using System.Windows.Input;
 
-namespace Microsoft.Toolkit.Uwp.SampleApp.Common
+namespace Microsoft.Toolkit.Uwp.Commands
```

Comment:
ICommand is under the `System.Windows.Input` namespace. Should this follow similar pattern? `Microsoft.Toolkit.Uwp.Input`